src/toil_scripts/lib/files.py

```diff
@@ -0,0 +1,79 @@
+from contextlib import closing
+import os
+import errno
+import tarfile
+import shutil
+
+
+# FIXME: replace with bd2k.util.files.mkdir_p
+def mkdir_p(path):
+    """
+    It is Easier to Ask for Forgiveness than Permission
+
+    :param str path: path to directory to create
+    """
+    try:
+        os.makedirs(path)
+    except OSError as exc:
+        if exc.errno == errno.EEXIST and os.path.isdir(path):
+            pass
+        else:
+            raise
+
+
+def tarball_files(tar_name, work_dir='.', prefix='', fpaths=None):
+    """
+    Creates a tarball from a group of files
+
+    :param str tar_name: Name of tarball
+    :param str work_dir: Current working directory
+    :param list[str] fpaths: File paths to include in the tarball
+    :param str prefix: Optional prefix for files in tarball
+    """
+    if fpaths is None:
+        fpaths = []
+    with tarfile.open(os.path.join(work_dir, tar_name), 'w:gz') as f_out:
+        for fpath in fpaths:
+            f_out.add(fpath, arcname=prefix + os.path.basename(fpath))
+
+
+def move_to_output_dir(output_dir, filepaths=None):
+    """
+    Moves files from the working directory to the output directory.
+
+    :param str output_dir: Output directory
+    :param list[str] filepaths: Filepaths to move
+    """
+    if filepaths is None:
```

Comment:
Who would call this method with None? Empty list yes, but None? If None is an actual use case, then make the for loop conditional rather than assigning empty list simply to skip the for loop.